components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesJobConverter.cs

```diff
@@ -232,8 +263,36 @@ public class PublishedNodesJobConverter {
                     }
                 }
             }
+
+            if (item.OpcEvents != null) {
+                foreach (var node in item.OpcEvents) {
+                    if (string.IsNullOrEmpty(node.Id)) {
+                        node.Id = node.ExpandedNodeId;
+                    }
+                    if (scaleTestCount == 1) {
+                        yield return node;
```

Comment:
Do I understand it correctly that we don't need to assign OpcPublishingInterval as on line #245 @simonjaeger

Reply:
Let me check @buzzfrog

Reply:
Fixed.